src/utils/transactions.ts

```diff
@@ -58,11 +58,12 @@ export const getTransactionAssetAmounts = (assetAmounts: AssetAmount[]) => {
     .filter((asset): asset is Required<AssetAmount> => asset.id !== ALPH.id && asset.amount !== undefined)
     .map((asset) => ({ id: asset.id, amount: asset.amount.toString() }))
 
-  const dustAmount = DUST_AMOUNT * BigInt(tokens.length)
-  const attoAlphAmount = (alphAmount + dustAmount).toString()
+  const minAlphAmountRequirement = DUST_AMOUNT * BigInt(tokens.length)
+  const minDiff = minAlphAmountRequirement - alphAmount
+  const totalAlphAmount = minDiff > 0 ? alphAmount + minDiff : alphAmount
 
   return {
-    attoAlphAmount,
+    attoAlphAmount: totalAlphAmount.toString(),
```

Comment:
@mvaivre I had to add this last commit. After the discussion here https://github.com/alephium/desktop-wallet/issues/682 I realized that we unnecessarily were forcing the user to send an additional 0.001 ALPH per token, no matter if they decide to send ALPH or not. I realized that the min requirement should be added **only if** the ALPH amount is not enough.

Sorry for breaking the code-freeze, but this was important to fix imo.